cicecore/cicedynB/general/ice_init.F90

```diff
@@ -384,10 +384,10 @@ subroutine input_data
       Ktens = 0.0_dbl_kind   ! T=Ktens*P (tensile strength: see Konig and Holland, 2010)
       e_yieldcurve = 2.0_dbl_kind ! VP aspect ratio of elliptical yield curve               
       e_plasticpot = 2.0_dbl_kind ! VP aspect ratio of elliptical plastic potential
-      visc_coeff_method = 'avg_strength' ! calc visc coeff at U point: avg_strength, avg_zeta
-      deltaminEVP = 1e-11_dbl_kind ! minimum delta for viscous coeff (EVP, Hunke 2001)
-      deltaminVP  = 2e-9_dbl_kind  ! minimum delta for viscous coeff (VP, Hibler 1979)
-      capping     = 1.0_dbl_kind   ! method for capping of visc coeff (1=Hibler 1979,0=Kreyscher2000)
+      visc_method = 'avg_strength' ! calc viscosities at U point: avg_strength, avg_zeta
+      deltaminEVP = 1e-11_dbl_kind ! minimum delta for viscosities (EVP, Hunke 2001)
```

Comment:
I think this should just be deltaminEVP = puny?

Reply:
This might disappear anyway...I think we should just have deltaminEVP = deltaminVP  = deltamin = 2e-9_dbl_kind

deltamin = 2e-9_dbl_kind is the original VP value and used in Bouillon 2013 and Kimmritz 2016.

To be discussed...